extensions/blocks/subscriptions/attributes.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const defaultAttributes = {
+	subscribePlaceholder: {
+		type: 'string',
+		default: __( 'Enter your email here', 'jetpack' ),
+	},
+	showSubscribersTotal: {
+		type: 'boolean',
+		default: false,
+	},
+	buttonOnNewLine: {
+		type: 'boolean',
+		default: false,
+	},
+	submitButtonText: {
+		type: 'string',
+		default: __( 'Sign Up', 'jetpack' ),
```

Comment:
When using `__()` it's important to test that saving the block + changing locale + reloading the editor doesn't bust these if they had translations. Not saying that would happen here, but just something to watch out for.

Reply:
Looks like it does bust. :/

Reply:
This seemed okay when changing languages.